.github/workflows/link-validation.yml

```diff
@@ -0,0 +1,14 @@
+name: "Check links with linkcheck"
+on:
+- pull_request
+
+jobs:
+  triage:
+    permissions:
+      contents: read
+    runs-on: ubuntu-latest
+    steps:
+    - uses: filiph/linkcheck@f2c15a0be0d9c83def5df3edcc0f2d6582845f2d #3.0.0
+      with:
+        arguments: -i https://docs.platform.uno
```

Comment:
@jeromelaban seems there is currently an error:
```
> Run filiph/linkcheck@f2c15a0be0d9c83def5df3edcc0f2d6582845f2d

Can't read input file 'https://docs.platform.uno': PathNotFoundException: Cannot open file, path = 'https://docs.platform.uno/' (OS Error: No such file or directory, errno = 2)
```

Reply:
@morning4coffe-dev will continue the work here.